packages/katello/katello/backup.rb

```diff
@@ -372,6 +372,12 @@ def validate_directory
       end
     end
 
+    def tape_length_validator(size)
+      with_specific_size = !!(size =~ /^\d+[bBcGKkMPTw]{1}$/)
+      in_bytes = !!(size =~ /^\d+$/)
+      with_specific_size || in_bytes
```

Comment:
I think this is the same as `size[/^\d+[bBcGKkMPTw]?$/]`

Reply:
@johnpmitsch, @akofink I think there is a difference. `{1}` means exactly one occurrence (same as no quantificator) while `?` means one or no occurrence. 
```
irb(main):003:0> !!('100' =~ /^\d+[bBcGKkMPTw]?$/)
=> true
irb(main):004:0> !!('100' =~ /^\d+[bBcGKkMPTw]{1}$/)
=> false
irb(main):005:0> !!('100' =~ /^\d+[bBcGKkMPTw]$/)
=> false
irb(main):006:0> !!('100M' =~ /^\d+[bBcGKkMPTw]$/)
=> true
```  
The original didn't allowed size without units. The new version does. While tar accepts no units as bytes I think the new version fits better. I'll update the tests in f-m to cover that.

Reply:
Pardon me, I obviously completely and embarrassingly missed the point. Forget my comment. ;)